cmd/cli/triggers.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"time"
+
+	"github.com/moira-alert/moira"
+)
+
+func handleRemoveTriggersByPrefix(logger moira.Logger, database moira.Database, prefix string) error {
+	triggers, err := database.GetTriggerIDsByPrefix(prefix)
+	if err != nil {
+		return err
+	}
+
+	// Added delay because command is potentially dangerous and can delete unwanted triggers
+	const delay = 10 * time.Second
+	logger.Infof("%d triggers by prefix %s would be removed after %s", len(triggers), prefix, delay)
+	logger.Info("You can cancel execution by Ctrl+C")
+	time.Sleep(delay)
+
+	logger.Infof("Removing triggers by prefix %s started", prefix)
+	for _, id := range triggers {
```

Comment:
pipe?

Reply:
from the [docs](https://redis.io/docs/manual/pipelining/): pipeline is used for improving performance. So pipe for this case is premature optimization, command works pretty fast